lib/components/PermitAndDepositUI.jsx

```diff
@@ -1,17 +1,14 @@
-import React, { useContext, useState } from 'react'
-import { ethers } from 'ethers'
-
-import CompoundPrizePoolAbi from '@pooltogether/pooltogether-contracts/abis/CompoundPrizePool'
-import PermitAndDepositDaiAbi from '@pooltogether/pooltogether-contracts/abis/PermitAndDepositDai'
 import DaiAbi from '@pooltogether/pooltogether-contracts/abis/Dai'
-
-import { signPermit } from 'lib/utils/signPermit'
-import { CONTRACT_ADDRESSES } from 'lib/constants'
+import PermitAndDepositDaiAbi from '@pooltogether/pooltogether-contracts/abis/PermitAndDepositDai'
```

Comment:
We should actually remove this line. We didn't end up using the Permit feature for security reasons.

Reply:
Looks like this component is used for mainnet dai, should that be changed?

Reply:
Yeah, let's rip out everything Permit. It's not worth the headache, doesn't work with hardware wallets, etc. etc.